op-supervisor/supervisor/backend/cross/safe_update.go

```diff
@@ -63,6 +67,11 @@ func CrossSafeUpdate(logger log.Logger, chainID eth.ChainID, d CrossSafeDeps) er
 	if err := d.UpdateCrossSafe(chainID, newScope, crossSafeRef); err != nil {
 		return fmt.Errorf("failed to update cross-safe head with L1 scope increment to %s and repeat of L2 block %s: %w", candidateScope, crossSafeRef, err)
 	}
+	m.RecordCrossSafeRef(eth.L2BlockRef{
+		Number: crossSafeRef.Number,
+		Time:   crossSafeRef.Time,
+		Hash:   crossSafeRef.Hash,
+	})
```

Comment:
Why not add the metrics update inside of the `db.UpdateCrossSafe` function? That way every time we call it, we update the metrics. Calling it separately here adds unnecessary complexity to the `CrossSafeUpdate` function.